public/js/visualisation_helpers.js

```diff
@@ -42,7 +42,8 @@ export function tick_formatter(scale, seq_type) {
     var suffixes = {amino_acid: 'aa', nucleic_acid: 'bp'};
 
     return function (d) {
-        return `${prefix(d)} ${suffixes[seq_type]}`;
+        return `${prefix(d)}${suffixes[seq_type]}`
+            .replace(/([0-9])([a-zA-Z])/g, '$1 $2');
```

Comment:
Minor, feel free to ignore: I think this would be simpler and achieve the same result?
```suggestion
            .replace(/([a-zA-Z]+)/, ' $1')
```

Reply:
@tadast yes you are right, code updated and replace code also added to length distribution